docs/src/chaos/periodicity.md

```diff
@@ -145,3 +145,6 @@ estimate_period(v, :periodogram, t)
 ```@example MAIN
 estimate_period(v, :zerocrossing, t)
 ```
+```@example MAIN
+estimate_period(v, :yin, t, sr=Int64(1/Δt), f0_min=0.01)
```

Comment:
word of advice: when calling a function with both positional arguments as well as keyword arguments, always separate the two with a `;`
```suggestion
estimate_period(v, :yin, t; sr=Int64(1/Δt), f0_min=0.01)
```

Reply:
Makes code clearer to read. 

Oh, also use `round(Int, (1/Δt)` instead. 

Oh, also, never use `Int64`. It is operating system specific. Always use just `Int`.

Reply:
All right, done. Thanks!